ns/rdf-xsd.ttl

```diff
@@ -0,0 +1,283 @@
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix owl:  <http://www.w3.org/2002/07/owl#> .
+@prefix dc:   <http://purl.org/dc/elements/1.1/> .
+
+<http://www.w3.org/2001/XMLSchema#> a owl:Ontology ;
+  dc:title "RDF XSD datatypes" ;
+  dc:date "2025-09-08" ;
+  dc:description "This file is a description of the XML Schema datatypes used in RDF" .
+
+xsd:string a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "string" ;
+  rdfs:comment "Character strings. Value space: finite-length sequences of Unicode characters. No length or whitespace normalization constraints by default." .
+
+xsd:boolean a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "boolean" ;
+  rdfs:comment "Two-valued logic. Lexical forms: 'true', 'false', '1', '0'. Value space: {true, false}." .
+
+xsd:decimal a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "decimal" ;
+  rdfs:comment "Arbitrary-precision decimal numbers. Value space: numbers of the form i/10^n with i and n integers (n ≥ 0). No special values (INF/NaN) and no exponential notation in lexical form." .
```

Comment:
`i` has special meaning in the world of mathematics. Perhaps a different character could be used here?

Reply:
The definition of [xsd;decimal in XMLSchema 1.1](https://www.w3.org/TR/xmlschema11-2/#decimal) uses `i` and `n`.

In computing, `i` is commonly an integer, often an array index.